src/main/kotlin/com/neva/gradle/fork/ForkExtension.kt

```diff
@@ -23,23 +27,27 @@ open class ForkExtension(val project: Project) {
   }
 
   fun config(configurer: Config.() -> Unit) {
-    config(SourceTargetConfig(project, Config.NAME_DEFAULT), configurer)
+    config(SourceTargetConfig(project, propertiesDefinitions, Config.NAME_DEFAULT), configurer)
```

Comment:
how about changing to

`SourceTargetConfig(this, Config.NAME_DEFAULT)`

the `ForkExtension` is already holding project, property definitions could be then public or internal. don't be afraid od passing these heavylooking objects. this will simplify code much

Reply:
OK